api/peering_test.go

```diff
@@ -205,6 +208,7 @@ func TestAPI_Peering_GenerateToken_Read_Initiate_Delete(t *testing.T) {
 
 		// require that the peering state is not undefined
 		require.Equal(r, PeeringStateInitial, respr.State)
+		require.Equal(r, map[string]string{"foo": "bar"}, resp2.Meta)
```

Comment:
I think this should be looking at `respr.Meta` and not `resp2.Meta`.

```suggestion
		require.Equal(r, map[string]string{"foo": "bar"}, respr.Meta)
```

Reply:
ah yes great catch!